src/vs/workbench/contrib/positronPlots/browser/components/webviewPlotInstance.tsx

```diff
@@ -14,6 +14,7 @@ interface WebviewPlotInstanceProps {
 	width: number;
 	height: number;
 	plotClient: WebviewPlotClient;
+	visible: boolean;
```

Comment:
This is needed for when the pane is hidden – we don't set `props.width = 0` in that case.